From 1993383ddf67e296334c7916d6afc699ee6300c7 Mon Sep 17 00:00:00 2001
Message-ID: <1993383ddf67e296334c7916d6afc699ee6300c7.1737052666.git.sam@gentoo.org>
From: Arun Raghavan <arun@asymptotic.io>
Date: Fri, 29 Nov 2024 10:54:18 -0500
Subject: [PATCH 1/8] spa: alsa: Fix key name suggestion in log message

We use api.alsa.pcm.card to look up the PCM's card if we don't know it
by other means.
---
 spa/plugins/alsa/alsa-pcm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spa/plugins/alsa/alsa-pcm.c b/spa/plugins/alsa/alsa-pcm.c
index b07897935..9244b66aa 100644
--- a/spa/plugins/alsa/alsa-pcm.c
+++ b/spa/plugins/alsa/alsa-pcm.c
@@ -939,7 +939,7 @@ int spa_alsa_init(struct state *state, const struct spa_dict *info)
 		sscanf(state->props.device, "%*[^:]:%u", &state->card_index);
 		if (state->card_index == SPA_ID_INVALID) {
 			spa_log_error(state->log, "Could not determine card index, maybe set %s",
-					SPA_KEY_API_ALSA_CARD);
+					SPA_KEY_API_ALSA_PCM_CARD);
 			return -EINVAL;
 		}
 	}
-- 
2.48.0

